sdk/core/azure-core/test/ut/transport_adapter_base_test.cpp

```diff
@@ -49,7 +49,9 @@ namespace Azure { namespace Core { namespace Test {
     auto request = Azure::Core::Http::Request(Azure::Core::Http::HttpMethod::Get, host);
     auto response = m_pipeline->Send(request, Context{});
     checkResponseCode(response->GetStatusCode(), Azure::Core::Http::HttpStatusCode::NoContent);
-    auto expectedResponseBodySize = std::stoull(response->GetHeaders().at("content-length"));
+    auto expectedResponseBodySize = 0;
```

Comment:
Can we make this check a bit stronger to say:
if responseCode == 204 then expectedResponseBodySize = 0
else
   auto expectedrbs = std:stoull..........

Reply:
Done.